src/common/components/buttons/IconButton.js

```diff
@@ -1,16 +1,16 @@
 import React from "react";
 import PropTypes from "prop-types";
-import { Knapp } from "@navikt/arbeidsplassen-knapper";
+import { Button } from "@navikt/ds-react";
 import "./IconButton.css";
 
 function IconButton({ text, icon, hideText, onClick, className, disabled, spinner, type }) {
     return (
-        <Knapp type={type} disabled={disabled} spinner={spinner} onClick={onClick} className={className}>
+        <Button variant="secondary" type={type} disabled={disabled} spinner={spinner} onClick={onClick} className={className}>
             <span className="IconButton__inner">
                 {icon}
```

Comment:
I den nye <Button> så er icon en egen prop. Så da blir det sånn `<Button icon={icon}`....>. Når du flytter icon inn der, så trenger du nok ikke `<span className="IconButton__inner">` lengre, siden Button nå ordner alt med ikon alignment for deg, så denne kan slettes.